docs/cmf_server/cmf-server.md

```diff
@@ -113,18 +119,50 @@ There are two ways to start cmf server -
    ```
    docker run --name mycontainer -p 0.0.0.0:8080:80 -v /home/user/cmf-server/data/static:/cmf-server/data/static myimage
    ```
-
-6. To stop the docker container.
+6. After cmf-server container is up, start `ui-server`, Go to `cmf/ui` folder.
+   ```
+   cd /cmf/ui
+   ```
+7. Execute the below-mentioned command to create a `ui-server` docker image.
+   ```
+   Usage:  docker build -t [image_name] -f ./Dockerfile ../
+   ```
+   Example:
+   ```
+   docker build -t uiimage -f ./Dockerfile ../
+   ```
+8. Launch a new docker container using the image with directory
+   <pre>
+   Usage: docker run --name [container_name] -p 0.0.0.0:3000:80 [image_name]
+   </pre>
+   Example:
+   ```
+   docker run --name mycontainer -p 0.0.0.0:3000:80 uiimage
+   ```
+   `Note` - If you face issue regarding `Libzar proxy` you need add proxy as follows:
```

Comment:
This note is not relying full information.

Reply:
Done